lib/async_hooks.js

```diff
@@ -49,6 +49,9 @@ const init_symbol = Symbol('init');
 const before_symbol = Symbol('before');
 const after_symbol = Symbol('after');
 const destroy_symbol = Symbol('destroy');
+const emitBeforeN = emitHookFactory(before_symbol);
```

Comment:
Question: is there a need to declare these? Stack beauty?
Otherwise could just be used in L#61